cdap-ui/app/features/dashboard/controllers/addwdgt-ctrl.js

```diff
@@ -65,7 +65,9 @@ function ($scope, $modalInstance, caskFocusManager, Widget) {
           new Widget({
             type: $scope.model.type,
             title: $scope.model.metric.title,
-            color: classes[(generateHash(value) * 13) % classes.length],
+            settings: {
+              color: classes[(generateHash(value) * 13) % classes.length]
```

Comment:
Hmm, actually, this can probably be removed.
Currently, this is how overview page's chart's colors are being set:
https://github.com/caskdata/cdap/blob/develop/cdap-ui/app/features/overview/overview.html#L84